Submitted By:            Igor Živković <contact at igor hyphen zivkovic dot from dot hr>
Date:                    2013-07-14
Initial Package Version: 2.0.6
Upstream Status:         Fixed
Origin:                  Upstream
Description:             Fixes compilation with FLAC 1.3.0

diff -Naur vlc-2.0.6.orig/modules/codec/flac.c vlc-2.0.6/modules/codec/flac.c
--- vlc-2.0.6.orig/modules/codec/flac.c	2013-01-25 13:30:14.000000000 +0100
+++ vlc-2.0.6/modules/codec/flac.c	2013-06-10 13:13:04.275642135 +0200
@@ -38,8 +38,8 @@
 #include <vlc_codec.h>
 #include <vlc_aout.h>
 
-#include <stream_decoder.h>
-#include <stream_encoder.h>
+#include <FLAC/stream_decoder.h>
+#include <FLAC/stream_encoder.h>
 
 #include <vlc_block_helper.h>
 #include <vlc_bits.h>
